.github/workflows/qa.yaml

```diff
@@ -12,3 +12,13 @@ jobs:
           REQUIRE_DEV: true
         with:
           args: analyse -c .phpstan/phpstan.neon.dist
+  psalm:
+    name: Psalm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Psalm
+        uses: docker://vimeo/psalm-github-actions
+        with:
+          args: --shepherd
```

Comment:
After merging, we got a type coverage report at:
https://shepherd.dev/github/sonata-project/SonataDoctrineORMAdminBundle

![Psalm coverage](https://shepherd.dev/github/sonata-project/SonataDoctrineORMAdminBundle/coverage.svg)

Reply:
This will be overriden by dev-kit. https://github.com/sonata-project/dev-kit/pull/1135/files

You should make the PR on dev-kit to add `--sheperd` args instead.

Reply:
Done